packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -44,6 +44,7 @@ export const messages = {
   parameter_gatherer_enter_file_name: 'Enter desired filename',
   parameter_gatherer_enter_dir_name:
     "Enter desired directory (Press 'Enter' to confirm or 'Escape' to cancel)",
+  parameter_gatherer_enter_username_name: 'Enter target username',
```

Comment:
@ruthemmanuelle This is the message that will show in the input box for the Display Org command where the user indicates which username. Target username is the name of the flag so I went with that.

Reply:
Just noticed that the message above this one includes directions about confirming/canceling. Do we want that to be uniform across all parameter gathering messages?

Reply:
Yeah, it looks like you added the message above this one as part of PR #47. I think it was while I was on PTO. Do you remember why you included the text in parentheses? Is it standard in VS Code? It would be good to be consistent.

Reply:
I think I added it after discussing with Nick. I've seen it in other extension commands like `Git: Create Branch...` but it doesn't seem to be a standard amongst commands.
![image](https://user-images.githubusercontent.com/4503716/30616019-aff0d1e4-9d45-11e7-867f-6be74b952d4b.png)

Reply:
Thanks for the info. Might as well go ahead and add it to the other prompts, and to this new one—it might be useful, and I don't think it will hurt.

Per Salesforce's style guidelines, Escape should be Esc, and there shouldn't be any quotation marks around Enter or Esc.